src/scripts/components/layer/wv.layer.list.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import LayerRadio from './wv.layer.radio.js';
+import { CellMeasurer, CellMeasurerCache, List } from 'react-virtualized';
+
+/*
+ * A react component, Builds a list of layers using the LayerRadio component
+ * @class LayerList
+ * @extends React.Component
+ */
+export default class LayerList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      layerFilter: props.config.layerOrder,
+      expandedLayers: []
+    };
+    this._cache = new CellMeasurerCache({
+      fixedWidth: true,
+      minHeight: 23,
+      minWidth: props.width
+    });
+  }
+
+  /*
+   * Clear cell catch if component will
```

Comment:
I don't think we should document this function. `componentWillUpdate` is a standard React lifecycle method, and this comment isn't really documenting what it does, or what the developers intent is here. What do you think about an inline comment where the cache is cleared to clarify what's happening?

```
this._cache.clearAll(); // Clear CellMeasurerCache
```

Really, a comment is only valuable here if it provides insight into why we're clearing the cache here (I don't know why we're doing that). But if we're going to have comments, they should document what we're doing, not what the libraries we're depending on do behind the scenes.